src/ubx.cpp

```diff
@@ -803,10 +800,6 @@ int GPSDriverUBX::configureDevice(const GPSConfig &config, const int32_t uart2_b
 		cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1INPROT_NMEA, 0, cfg_valset_msg_size);
 		cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1OUTPROT_UBX, 1, cfg_valset_msg_size);
 		cfgValset<uint8_t>(UBX_CFG_KEY_CFG_UART1OUTPROT_RTCM3X, 1, cfg_valset_msg_size);
-
-		cfgValset<uint8_t>(UBX_CFG_KEY_MSGOUT_UBX_NAV_RELPOSNED_UART1, 0, cfg_valset_msg_size);
```

Comment:
Just to double-check: as it's now enabled on both devices, are you getting this message only on 1 device (the rover)?

Reply:
I will have to trace it if you really want exact answer, but my understanding is that only the main ("rover") device will compute heading, and send it, while the secondary ("moving base") doesn't do it, as it only gets RTCM from the land base and doesn't have info about the second antenna location. I might be wrong. Is it important if messages are sent from both, as only the Main GPS is passing them to ORB?

Reply:
OK, I traced it and I think I am finally getting what you meant. The RELPOSNED was coming from both devices, but was ignored unless it was a "Rover with moving base" (ubx.cpp : 2155). So, I think it would be proper to only set it up under the same condition (my latest check-in ubx.cpp : 668).

For HPPOSLLH we better allow it for all cases, as it is beneficial to see actual coordinates of both antennas, using "gps status".

Sounds right?

Reply:
Ok thanks for checking. Yes that was my concern.